src/webgpu/shader/execution/limits.spec.ts

```diff
@@ -0,0 +1,438 @@
+export const description = `Execution tests for WGSL limits.`;
+
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { keysOf } from '../../../common/util/data_tables.js';
+import { iterRange } from '../../../common/util/util.js';
+import { GPUTest } from '../../gpu_test.js';
+import { CheckElementsGenerator, checkElementsEqualGenerated } from '../../util/check_contents.js';
+
+export const g = makeTestGroup(GPUTest);
+
+// The limits that we test.
+const kMaxStructMembers = 1024;
+const kMaxCompositeNestingDepth = 16;
+const kMaxBraceNestingDepth = 63;
```

Comment:
issue: This variable and its usage appear to be intended to resolve <https://github.com/gpuweb/cts/issues/1956>, but the spec. currently specifies a limit of 127. Isn't this value (`63`) too low?

Reply:
Yeah I've put it back to `127` (the `63` was from a tentative spec change proposal that I didn't pursue).